vastloader.go

```diff
@@ -0,0 +1,59 @@
+package vast
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"time"
+)
+
+type VASTResponse struct {
+	Body []byte
+	Err  error
+}
+
+// VASTResponseHandler handles the repsonse from the request
+func ResponseHandler(res *http.Response, err error) VASTResponse {
```

Comment:
```suggestion
func ResponseHandler(res *http.Response, err error) ([]byte, error) {
```
Really nit-picky, but I think returning multiple values gives whomever uses this library the flexibility to decide how they want to use (or not use) the values

Reply:
Ah… that was a byproduct of when I used a channel earlier.